.circleci/config.yml

```diff
@@ -438,6 +450,18 @@ jobs:
           bazel run @graknlabs_build_tools//ci:release-validate-deps -- \
             graknlabs_common graknlabs_graql graknlabs_protocol graknlabs_console
```

Comment:
This should be updated to take in `@graknlabs_console_distribution` now, and make sure it still works.